src/vue/pages/Signup.vue

```diff
@@ -30,18 +30,35 @@
         <div class="signup__seed-wrp">
           <p class="signup__seed-explanations">
             {{ 'auth-pages.save-recovery-seed-details' | globalize }}
+            <b>
```

Comment:
use markdown instead of concatenating different messages to apply different styles for them